org.uqbar.project.wollok.lib/src/wollok/lang/WObject.xtend

```diff
@@ -10,6 +10,7 @@ import org.uqbar.project.wollok.interpreter.core.ToStringBuilder
 import org.uqbar.project.wollok.interpreter.core.WollokObject
 import org.uqbar.project.wollok.sdk.WollokDSK
 
+import static extension org.uqbar.project.wollok.errorHandling.HumanReadableUtils.*
```

Comment:
This added import looks strange as there is no other change in the file, is it used?

Reply:
Yes, because I moved an extension method from other extension import to that class